lib/components/Checkbox/Checkbox.docs.tsx

```diff
@@ -151,14 +151,14 @@ const docs: ComponentDocs = {
         source(
           <Stack space="medium">
             <Checkbox
-              id={id}
+              id={`${id}_unchecked`}
               disabled={true}
               checked={false}
-              onChange={() => toggleState('checked')}
+              onChange={() => toggleState('unchecked')}
```

Comment:
Nitpick: feel like this should stay as `checked`. Not that the state name matters too much, but I think it makes more sense to match the name of the prop so you don't have to negate it

Reply:
There are two Checkboxes here. Having both with `checked` means they are sharing state.